simulation/simple_sensor_simulator/test/src/vehicle_simulation/test_ego_entity_simulation.cpp

```diff
@@ -40,19 +40,20 @@ TEST(EgoEntitySimulation, calculateAccelerationBySlope)
   // expected value in the lanelet(id:7)
   // first 25m: 1m up
   constexpr double expected_slope_acceleration_first_25m =
-    std::sin(std::atan(1. / 25.)) * gravity_acceleration;
+    -std::sin(-std::atan(1. / 25.)) * gravity_acceleration;
```

Comment:
Please use [std::atan2](https://cpprefjp.github.io/reference/cmath/atan2.html) instead of std::atan

Reply:
I fixed in [`cee4c42` (#1477)](https://github.com/tier4/scenario_simulator_v2/pull/1477/commits/cee4c4244f660aee30000126446b78133edf92ab)